### ISSUE TITLE
duplicate chunk folder on gcd / google drive

### ISSUE THREAD

**Issue by tfriedel** [gilbertchen/duplicacy#585]
I created an initial backup to google drive and tried to restore it. After a couple of chunks I see the message "Chunk ace650f76d28f021eac474012a398490cf0241c1a7fee43ae3e984c182819dbe can't be found" and the restore process is aborted. Manually inspecting the google drive "cunks" folder I see there's two folder that start with "ac":
ac
ac (1)
I have no explanation for this, but suspect it's a bug and there should only be one folder.
I should maybe mention that I have concurrent backups running from 3 computers at the same time (2 linux, one windows).

**Comment by gilbertchen** [comment]
Yes, google drive does allow two folders or files to have the same name.  They use ids to uniquely identify each file/folder.

This bug is caused by multiple computers trying to create the same folder at the same time.  Ideally, Duplicacy should create all those 256 hex subfolders when initializing the storage.

As a workaround you can manually move all chunks under folders with the same name and remove the empty ones.

**Comment by fingerwarrior** [comment]
Any update on this? just got bit by the same issue when duplicacy created two different `cc` folders and any further checks failed to find the chunks in the second folder. I have only one computer backing up to the drive though, with parallel jobs turned off in schedules, so not sure what triggered the creation of duplicate `cc` folder. Moving the files to the correct folder and deleting the other cc folder made the check pass.

I do have another backup running that is doing the initial upload. Is it possible that duplicacy did not "know" that another cc folder already exists because no snapshot had a referenced chunk to the cc folder and just created a duplicate cc folder?
